Extension/src/LanguageServer/extension.ts

```diff
@@ -462,12 +462,22 @@ function realActivation(): void {
 
     vcpkgDbPromise = initVcpkgDatabase();
 
-    if (settings.updateChannel === 'Default') {
-        suggestInsidersChannel();
-    } else if (settings.updateChannel === 'Insiders') {
-        insiderUpdateTimer = global.setInterval(checkAndApplyUpdate, insiderUpdateTimerInterval, settings.updateChannel);
-        checkAndApplyUpdate(settings.updateChannel);
-    }
+    PlatformInformation.GetPlatformInformation().then(info => {
+        // Skip Insiders processing for 32-bit Linux.
+        if (info.platform !== "linux" || info.architecture === "x86_64") {
```

Comment:
This check should probably move to `checkAndApplyUpdate`.  Otherwise this logic needs to exist both here and in `onDidChangeSettings`.  I'd rather it be in just one place.

Reply:
Umm...but it needs to surround suggestInsidersChannel too. I'll look into onDidChangeSettings.

Reply:
I just added an enabled Boolean.